packages/babel-plugin/__tests__/stylex-transform-define-consts-test.js

```diff
@@ -0,0 +1,354 @@
+          [
+            "x1izlsax",
+            {
+              "constKey": "sm",
+              "constVal": "(min-width: 768px)",
+              "ltr": "",
+              "rtl": null,
+            },
+            0,
+          ],
+          [
+            "xe5hjsi",
+            {
+              "constKey": "md",
+              "constVal": "(min-width: 1024px)",
+              "ltr": "",
+              "rtl": null,
+            },
+            0,
+          ],
+          [
+            "xmbwnbr",
+            {
+              "constKey": "lg",
+              "constVal": "(min-width: 1280px)",
+              "ltr": "",
+              "rtl": null,
+            },
+            0,
+          ],
+        ]
+      `);
+    });
+
+    test('transforms constants object with import *', () => {
+      const { code, metadata } = transform(`
+        import * as foo from 'stylex';
+        export const colors = foo.defineConsts({
+          primary: '#ff0000',
+          secondary: '#00ff00',
+          tertiary: '#0000ff',
+        });
+      `);
+
+      expect(code).toMatchInlineSnapshot(`
+        "import * as foo from 'stylex';
+        export const colors = {
+          primary: "#ff0000",
+          secondary: "#00ff00",
+          tertiary: "#0000ff"
+        };"
+      `);
+
+      expect(metadata.stylex).toMatchInlineSnapshot(`
+        [
+          [
+            "xbx9tme",
+            {
+              "constKey": "primary",
+              "constVal": "#ff0000",
+              "ltr": "",
+              "rtl": null,
+            },
+            0,
+          ],
+          [
+            "x1is3lfz",
+            {
+              "constKey": "secondary",
+              "constVal": "#00ff00",
+              "ltr": "",
+              "rtl": null,
+            },
+            0,
+          ],
+          [
+            "x1uyqs0n",
+            {
+              "constKey": "tertiary",
+              "constVal": "#0000ff",
+              "ltr": "",
+              "rtl": null,
+            },
+            0,
+          ],
+        ]
+      `);
+    });
+
+    test('handles special characters in constant names', () => {
+      const { code, metadata } = transform(`
+        import stylex from 'stylex';
+        export const sizes = stylex.defineConsts({
+          'max-width': '1200px',
+          'font-size*large': '18px',
+        });
+      `);
+
+      expect(code).toMatchInlineSnapshot(`
+        "import stylex from 'stylex';
+        export const sizes = {
+          "max-width": "1200px",
+          "font-size*large": "18px"
+        };"
+      `);
+
+      expect(metadata.stylex).toMatchInlineSnapshot(`
+        [
+          [
+            "x1b5i5wj",
+            {
+              "constKey": "max-width",
+              "constVal": "1200px",
+              "ltr": "",
+              "rtl": null,
+            },
+            0,
+          ],
+          [
+            "x4spo47",
+            {
+              "constKey": "font-size*large",
```

Comment:
the const key should be escaped to only use valid characters for CSS.

Reply:
The usage of such a key will probably be escaped when it is referenced within a `stylex.create`.